Documentation/AssociationsBasics.md

```diff
@@ -1338,8 +1338,8 @@ This code compiles, but you'll get a runtime fatal error "Not implemented: chain
 ```swift
 // NOT IMPLEMENTED
 let request = Book
-    .including(Book.author) // key "author"
-    .including(Book.author) // key "author"
+    .including(required: Book.author) // key "author"
```

Comment:
Not sure about this one. Might it was omitted on purpose?

Reply:
You're right, @kluufger